packages/governance/tools/fakeContractGovernor.js

```diff
@@ -0,0 +1,95 @@
+import '../src/types-ambient.js';
+
+import { E } from '@endo/eventual-send';
+import { Far } from '@endo/marshal';
+
+// eslint-disable-next-line no-unused-vars
+import { CONTRACT_ELECTORATE } from '../src/contractGovernance/governParam.js';
+
+// @file fakeContractGovernor is a simplified copy of ContractGovernor to
+// simplify testing. It has no electorate, and doesn't try to support
+// legibility. It does try to ensure that the governed contract can get complete
+// support for governance of parameters, apis, and filters. Tests should be able
+// to update parameters directly.
+
+/**
+ * @template {() => {creatorFacet: GovernorFacet<any>, publicFacet: GovernedPublicFacetMethods} } SF Start function of governed contract
+ * @param {ZCF<{
+ *   timer: TimerService,
+ *   governedContractInstallation: Installation<SF>,
+ *   governed: {
+ *     issuerKeywordRecord: IssuerKeywordRecord,
+ *     terms: {governedParams: {[CONTRACT_ELECTORATE]: Amount<'set'>}},
+ *     privateArgs: Record<string, unknown>,
```

Comment:
IDT this is right because https://github.com/Agoric/agoric-sdk/blob/91134411c97c4c120c25aa21aa645a986468046d/packages/governance/src/contractGovernor.js#L72-L73

```suggestion
```
Now seems like a good time to fix that upstream too, 
https://github.com/Agoric/agoric-sdk/blob/91134411c97c4c120c25aa21aa645a986468046d/packages/governance/src/contractGovernor.js#L119
https://github.com/Agoric/agoric-sdk/blob/91134411c97c4c120c25aa21aa645a986468046d/packages/governance/src/contractGovernor.js#L132

Reply:
I must be mis-reading something here. I see `privateArgs: Record<string, unknown>` as the old and new version here, and it matches what I already see at both locations in `contractGovernor.js`. What am I not seeing?

Reply:
That's the right type for `privateArgs`. The problem is it being under the `terms` parameter (redundantly). It should only be under `privateArgs`, right? See c542cb6eaa52b131c589ad5197373276db6c2452

Reply:
Okay, I think this is correct now.